pipeline.py

```diff
@@ -731,14 +731,6 @@ def change_dsl_function_to_normal_function(rendered_code: list):
 
     import re
 
-    # Regular expression to match ".path" but not "os.path"
```

Comment:
Sorry it looks like this is still present in https://github.com/redhat-et/ilab-on-ocp/pull/155/files/f24a5938e1ffb93a69f19096a031d53eb4e6d2a3..a588ed0ac06ef8478a553f74341703888f2b676b#diff-624e4842e8ca66f1dd2807228518a03ceb3a00b566f2e3ce4e073be8377bfae4R2065 so we still need this code.

Reply:
Reverted. 🙂